recipes/mtbseq/meta.yaml

```diff
@@ -0,0 +1,40 @@
+package:
+  name: mtbseq
+  version: "1.0.2"
```

Comment:
Please move the version into a jinja2 variable and use it in the source url. This makes an update of the recipe easier.

Reply:
Changed to jinja2 variable.